datadog_lambda/tags.py

```diff
@@ -35,16 +35,27 @@ def parse_lambda_tags_from_arn(arn):
     split_arn = arn.split(":")
 
     # If ARN includes version / alias at the end, drop it
-    if len(split_arn) > 7:
-        split_arn = split_arn[:7]
+    if len(split_arn) <= 7:
```

Comment:
Does the ARN ever not contain the version/alias? This might not be necessary if it always has it.